src/status_im/ui/screens/discover/all_dapps/views.cljs

```diff
@@ -44,6 +45,19 @@
                       :content-container-style           styles/dapp-preview-flat-list}]
      [react/text (i18n/label :t/none)])])
 
+(def columns 3)
```

Comment:
Why not put this in a let for main and pass it as arg to add-blank-dapps-for-padding?

Reply:
I initially didn't want to assign it every time the screen is run, since it's really a constant. But it sure looks more functional-ish that way, I agree. Done